# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1247574004 -7200
# Node ID 4088df5ec2e06e4ced5a7e05043329ce9ea151a6
# Parent  67d4c334134bd4d457f52bbf38c0feeb3f5b15f2
remove xhtml by default and also try and fail with sendscript using with-js macro

diff -r 67d4c334134b -r 4088df5ec2e0 src/page-template.lisp
--- a/src/page-template.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/page-template.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -46,7 +46,7 @@
 							   '((:script "weblocks-debug")
 							     (:stylesheet "debug-toolbar"))))))))
     (with-html-output (*weblocks-output-stream* nil :prologue t)
-      (:html :xmlns "http://www.w3.org/1999/xhtml"
+      (:html ;:xmlns "http://www.w3.org/1999/xhtml"
 	     (:head
 	      (:title (str (page-title app)))
 	      (render-page-headers app)
diff -r 67d4c334134b -r 4088df5ec2e0 src/snippets/html-utils.lisp
--- a/src/snippets/html-utils.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/snippets/html-utils.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -352,8 +352,8 @@
       (*defer-inline-js*
        (locally
          (declare (special *page-scripts*))
-         (push-end (with-javascript-to-string script)
-		   ;script
+         (push-end ;(with-javascript-to-string script)
+		   script
 		   *page-scripts*)))
       (t
        (with-javascript
# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1247574004 -7200
# Node ID 4088df5ec2e06e4ced5a7e05043329ce9ea151a6
# Parent  67d4c334134bd4d457f52bbf38c0feeb3f5b15f2
remove xhtml by default and also try and fail with sendscript using with-js macro

diff -r 67d4c334134b -r 4088df5ec2e0 src/page-template.lisp
--- a/src/page-template.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/page-template.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -46,7 +46,7 @@
 							   '((:script "weblocks-debug")
 							     (:stylesheet "debug-toolbar"))))))))
     (with-html-output (*weblocks-output-stream* nil :prologue t)
-      (:html :xmlns "http://www.w3.org/1999/xhtml"
+      (:html ;:xmlns "http://www.w3.org/1999/xhtml"
 	     (:head
 	      (:title (str (page-title app)))
 	      (render-page-headers app)
diff -r 67d4c334134b -r 4088df5ec2e0 src/snippets/html-utils.lisp
--- a/src/snippets/html-utils.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/snippets/html-utils.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -352,8 +352,8 @@
       (*defer-inline-js*
        (locally
          (declare (special *page-scripts*))
-         (push-end (with-javascript-to-string script)
-		   ;script
+         (push-end ;(with-javascript-to-string script)
+		   script
 		   *page-scripts*)))
       (t
        (with-javascript
# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1247574004 -7200
# Node ID 4088df5ec2e06e4ced5a7e05043329ce9ea151a6
# Parent  67d4c334134bd4d457f52bbf38c0feeb3f5b15f2
remove xhtml by default and also try and fail with sendscript using with-js macro

diff -r 67d4c334134b -r 4088df5ec2e0 src/page-template.lisp
--- a/src/page-template.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/page-template.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -46,7 +46,7 @@
 							   '((:script "weblocks-debug")
 							     (:stylesheet "debug-toolbar"))))))))
     (with-html-output (*weblocks-output-stream* nil :prologue t)
-      (:html :xmlns "http://www.w3.org/1999/xhtml"
+      (:html ;:xmlns "http://www.w3.org/1999/xhtml"
 	     (:head
 	      (:title (str (page-title app)))
 	      (render-page-headers app)
diff -r 67d4c334134b -r 4088df5ec2e0 src/snippets/html-utils.lisp
--- a/src/snippets/html-utils.lisp	Mon Jul 13 16:12:07 2009 +0200
+++ b/src/snippets/html-utils.lisp	Tue Jul 14 14:20:04 2009 +0200
@@ -352,8 +352,8 @@
       (*defer-inline-js*
        (locally
          (declare (special *page-scripts*))
-         (push-end (with-javascript-to-string script)
-		   ;script
+         (push-end ;(with-javascript-to-string script)
+		   script
 		   *page-scripts*)))
       (t
        (with-javascript
